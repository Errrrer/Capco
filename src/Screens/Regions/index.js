import React, {useState,useEffect} from 'react'
import { View, Text} from 'react-native'

const Index = ({nevigation}) =>{

  return (
     <View>
      <Text testID={'textTest'}>Regions Page</Text>
     </View>
  )
}

export default Index;