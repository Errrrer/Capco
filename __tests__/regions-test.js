import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';

import Regions from '../src/Screens/Regions';


const page = renderer.create(<Regions/>)

test('render regions page', () => {
    expect(page).toMatchSnapshot();
});

test('render text', () => {
   const text = page.root.findByProps({testID:'textTest'}).props;
   expect(text.children).toEqual('Regions Page');
})