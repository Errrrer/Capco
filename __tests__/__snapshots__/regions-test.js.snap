// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render regions page 1`] = `
<View>
  <Text
    testID="textTest"
  >
    Regions Page
  </Text>
</View>
`;
